### PR TITLE
Fix colony admin permissions dialog

### PR DESCRIPTION
**Pull request by sprusr**
## Description

Various fixes for DLP phase 1.

**Changes** 🏗

* It wasn't possible to unset roles
* It wasn't possible to set admin role
* Role names are now from `react-intl` messages

**Deletions** ⚰️

* Removed an unwanted console.log

### CODE REVIEW

File: src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx [hunk 1]
```diff
@@ -149,44 +176,52 @@ const ColonyPermissionEditDialog = ({
...
-          if (checkIfCanBeSet(role)) {
-            accumulator[role.toUpperCase()] = true;
-            return accumulator;
-          }
-          return accumulator;
-        }, {}),
+        domainId: domain.id,
+        colonyAddress,
+        roles: availableRoles.reduce(
+          (acc, role) => ({ ...acc, [role]: !!p.roles.find(r => r === role) }),
```
Comment (JamesLefrere) [inline src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx]: `p.roles.includes(role)` ?

File: src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx [hunk 2]
```diff
@@ -149,44 +176,52 @@ const ColonyPermissionEditDialog = ({
...
-  /* Currently arbitration should appear in the list of roles
-    but can not be set yet
-   */
-  const checkIfCanBeSet = role => {
-    if (role === 'Arbitration' || role === 'Root') {
-      return false;
-    }
-    return true;
-  };
+  // Get the current user's roles in the selected domain
```
Comment (JamesLefrere) [inline src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx]: Same :-/

Comment (sprusr) [reply src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx]: Huh?

Comment (JamesLefrere) [reply src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx]: Huh, that appeared on the wrong line... I was rather attempting to comment about the enum.

**Pull request merged**